SPECS/ipset/ipset.spec

```diff
@@ -34,12 +34,13 @@ Libraries and header files for ipset.
 
 %package service
 Summary:        %{name} service for %{name}s
-BuildRequires:  systemd
+BuildRequires:  systemd-bootstrap-rpm-macros
 Requires:       %{name} = %{version}-%{release}
 Requires:       iptables-services
-Requires(post): systemd
-Requires(postun): systemd
-Requires(preun): systemd
+# Our build tooling cannot handle this
+#Requires(post): systemd
+#Requires(postun): systemd
+#Requires(preun): systemd
```

Comment:
What error were you getting with these dependencies enabled? With the `BuildRequires` change above, I think there should be no issues with circular dependencies for this spec.

Having said, that I think I am leaning towards treating such `systemd` run-time dependencies as a special case. Like you noticed, I also can't image a running VM image without `systemd`. Containers on the other hand and our build environment can't make use of `systemd` services and this is what these run-time dependencies are about. In other words, dropping a dependency on `systemd` for cases where it's only there because a package wants to run a service should not have any negative side-effects but would simplify building our packages.

Anyway, I'm fine with commenting out these dependencies for this PR. We may need to think this through, but I'm curious, if you agree with this assessment?

Reply:
I am not an expert in our tooling, so I can't answer specifically why our tooling fails due to this, but it does.

> Containers on the other hand and our build environment can't make use of systemd services and this is what these run-time dependencies are about.

Er, I don't think that's actually correct, but in any case it seems we're in agreement that we can comment these out, at least for now.